filters/filter_17_Chinese/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+  "filterId": 224,
+  "name": "AdGuard Chinese filter",
+  "description": "EasyList China + AdGuard Chinese filter. Filter list that specifically removes ads on websites in Chinese language.",
+  "timeAdded": 1588759046083,
+  "homepage": "https://kb.adguard.com/en/general/adguard-ad-filters#chinese-filter",
+  "expires": "4 days",
+  "displayNumber": 1,
+  "groupId": 7,
+  "subscriptionUrl": "https://filters.adtidy.org/extension/chromium/filters/17.txt",
```

Comment:
`../224.txt` should be here (as filterId value), isn't it?

Reply:
Yes

Reply:
Changed https://github.com/AdguardTeam/FiltersRegistry/pull/333/commits/222b95eeecb343c6c31a645bf18bc4f94dbe2de6